src/elvis_style.erl

```diff
@@ -394,14 +394,27 @@ dont_repeat_yourself(Config, Target, RuleConfig) ->
 max_module_length(Config, Target, RuleConfig) ->
     MaxLength = maps:get(max_length, RuleConfig, 500),
     IgnoreModules = maps:get(ignore, RuleConfig, []),
+    CountComments = maps:get(count_comments, RuleConfig, false),
```

Comment:
I tried that first (actually with `skip_*`) but it is a lot more confusing for the user when setting a value for the options. `count_*` is an affirmation while `ignore_*` is a negation. What should `ignore_comments` do? Ignore them in the sense of counting them or ignore them in the sense of not counting them?